LogcatLib/src/main/java/info/hannes/logcat/base/LogListAdapter.kt

```diff
@@ -26,7 +26,7 @@ class LogListAdapter(private val completeLogs: MutableList<String>, filter: Stri
         filterLogs = completeLogs.filter { line ->
             var include = false
             for (filter in filters)
-                if (!include && line.contains(filter))
+                if (!include && line.contains(filter, true))
```

Comment:
@1951FDG 
It needs an improvement https://github.com/AppDevNext/Logcat/pull/60/files for this error https://github.com/owncloud/android-library/pull/330#issuecomment-653653819

But now I hope it works properly